mailpoet/tests/performance/tests/lists-view-subscribers.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-default-export */
+/**
+ * External dependencies
+ */
+import { sleep, check } from 'k6';
+import { chromium } from 'k6/experimental/browser';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+  baseURL,
+  thinkTimeMin,
+  thinkTimeMax,
+  headlessSet,
+  timeoutSet,
+  defaultListName,
+} from '../config.js';
+import { authenticate } from '../utils/helpers.js';
+/* global Promise */
+
+export async function listsViewSubscribers() {
+  const browser = chromium.launch({
+    headless: headlessSet,
+    timeout: timeoutSet,
+  });
+  const page = browser.newPage();
+
+  try {
+    // Go to the page
+    await page.goto(
+      `${baseURL}/wp-admin/admin.php?page=mailpoet-segments#/lists`,
+      {
+        waitUntil: 'networkidle',
+      },
+    );
+
+    // Log in to WP Admin
+    authenticate(page);
+
+    // Wait for async actions
+    await Promise.all([page.waitForNavigation({ waitUntil: 'networkidle' })]);
```

Comment:
Given that this has a single action, you could do just this instead:
```js
await page.waitForNavigation({ waitUntil: 'networkidle' });
```

Reply:
It's functionally identical, so no need to change now. I approved the PR already.

Reply:
Yes but I think the test keep failing if I don't put Promise.all, I will try once again and if it works without will place it. However, I am working on a new logging method so this will be probably deleted.